src/external/rawspeed/rawspeed-identify.cpp

```diff
@@ -119,7 +119,11 @@ int main(int argc, const char* argv[])
 
     fprintf(stderr, "Loading file: \"%s\"\n", argv[1]);
 
+    #if defined (_WIN32)
```

Comment:
Let me reword. We don't define either one of those (or do we?)
What does work with `_WIN32` that doesn't work with `WIN32`?